ui/ozone/platform/starboard/test/starboard_test_helper.h

```diff
@@ -46,11 +55,19 @@ class TestPlatformDelegate : public ui::PlatformWindowDelegate {
   void OnAcceleratedWidgetDestroyed() override {}
   void OnActivationChanged(bool active) override {}
   void OnMouseEnter() override {}
-
-  ui::EventType GetEventType() { return type; }
+  int bounds_changed_count() const { return bounds_changed_count_; }
+  bool closed() const { return closed_; }
+  ui::Event* dispatched_event() const { return dispatched_event_; }
```

Comment:
nit: Is this used? If not we can remove this accessor and the underlying member.